lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -170,8 +171,10 @@ def decorated(*args, db: DatabaseSession, log_output: bool, **kwargs):
                 # Records with repeating instruments or longitudinal
                 # events will have multiple entries in the list.
                 redcap_records: DefaultDict[str, List[dict]] = defaultdict(list)
-                for record in project.records(ids = record_ids, raw = raw_coded_values):
-                    redcap_records[record.id].append(record)
+
+                for chunk in chunked(record_ids, 5000):
```

Comment:
Small request to make this magic number a global variable or comment on it

Reply:
This number is now an option default, where it is self-documenting.